pages/rdm_cycle/sharing.md

```diff
@@ -1,40 +1,47 @@
 ---
 title: Sharing
+contributors: [Flora D'Anna, Bert Droesbeke, Niclas Jareborg]
 keywords:
 ---
 
 ## What is Data Sharing?
-Data Sharing datasets means making datasets known and available to the whole world and society.
-Does not have to happen immediately, while you are still working on the project. Data can be shared after your scientific paper is published.
+Data Sharing means making datasets known and available to the global research community and to society at large. Data sharing can be done at any time during the research data life cycle, but at the latest, data should be made available at the time of publication of articles that use the data to make scientific conclusions. 
```

Comment:
I would move the comma to after but, i.e. '...data life cycle but**,** at the latest, ...'